scripts/tests.sh

```diff
@@ -70,58 +69,58 @@ then
     fi
 fi
 
-function download_eth()
+function download_aleth()
 {
     if [[ "$OSTYPE" == "darwin"* ]]; then
-        ETH_PATH="$REPO_ROOT/eth"
+        ALETH_PATH="$REPO_ROOT/aleth"
     elif [ -z $CI ]; then
-        ETH_PATH="eth"
+        ALETH_PATH="aleth"
     else
         mkdir -p /tmp/test
         if grep -i trusty /etc/lsb-release >/dev/null 2>&1
         then
             # built from d661ac4fec0aeffbedcdc195f67f5ded0c798278 at 2018-06-20
-            ETH_BINARY=aleth_2018-06-20_trusty
-            ETH_HASH="54b8a5455e45b295e3a962f353ff8f1580ed106c"
+            ALETH_BINARY=aleth_2018-06-20_trusty
+            ALETH_HASH="54b8a5455e45b295e3a962f353ff8f1580ed106c"
         else
             # built from d661ac4fec0aeffbedcdc195f67f5ded0c798278 at 2018-06-20
-            ETH_BINARY=aleth_2018-06-20_artful
-            ETH_HASH="02e6c4b3d98299885e73f7db6c9e3fbe3d66d444"
+            ALETH_BINARY=aleth_2018-06-20_artful
+            ALETH_HASH="02e6c4b3d98299885e73f7db6c9e3fbe3d66d444"
         fi
-        wget -q -O /tmp/test/eth https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/$ETH_BINARY
-        test "$(shasum /tmp/test/eth)" = "$ETH_HASH  /tmp/test/eth"
+        ALETH_PATH="/tmp/test/aleth"
+        wget -q -O $ALETH_PATH https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/$ALETH_BINARY
+        test "$(shasum $ALETH_PATH)" = "$ALETH_HASH  $ALETH_PATH"
```

Comment:
There was a typo here previously the uppercase L was lowercase, hence the failures.